cpp/util/etcd.cc

```diff
@@ -997,9 +1011,9 @@ void EtcdClient::MaybeLogEtcdVersion() {
   }
   logged_version_ = true;
 
-  const UrlFetcher::Request req(
-      URL("http://" + etcds_.front().first + ":" +
-              to_string(etcds_.front().second) + "/version"));
+  const UrlFetcher::Request req(URL("http://" + etcds_.front().first + ":" +
```

Comment:
It seems to have done the opposite with this than with some of the other changes.